docs(dw): update cryptoasset page

sleepdefic1t opened the pull request:
## Summary

- use relative doc links
- resolve markdown lint warning ln#20
- remove trailing white space
- add AVAX to coins list
- add DOT to coins list
- add EGLD to coins list
- add SOL to coins list

## Checklist

- [x] Ready to be merged

faustbrian commented on docs/cookbook/developer/send-transaction-ark-sdk.md:
```diff
@@ -0,0 +1,221 @@
+
+const transaction = transactionBuilder.transfer()
+  .vendorField("something")
+  .recipientId(crypto.getAddress(recipientKeys.publicKey))
```
I would change this to `.recipientId('some ark address')` so new people don't think they need the passphrase of the recipient.

mvega3 replied:
Yeah. I specified that they didn't need this passphrase in the text above and wrote about why I did it this way, but it does feel weird anyways. I'll find another way to do it.

faustbrian replied:
Yeah, a lot of people will probably just fly over the code samples and might end up skipping some of the text so the code should be as clear as possible without additional text.

faustbrian commented on docs/cookbook/developer/send-transaction-ark-sdk.md:
```diff
@@ -0,0 +1,221 @@
+testnetClient
+  .resource('transactions')
+  .create({ transactions: [transaction] })
+  .then(res => console.log(res))
```
I would change this to `.then(res => console.log(res.data))` as the current code will log the whole axios response object instead of just the JSON like in the example below.

Commit:
docs(dw): update cryptoasset page

- use relative doc links
- resolve markdown lint warning ln#20
- remove trailing white space
- add AVAX to coins list
- add DOT to coins list
- add EGLD to coins list
- add SOL to coins list
